website/versioned_docs/version-v2.0_alpha/use-cases/public-goods-funding/quadratic-funding/how-it-works.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Public Goods Funding - How it works
+description: Overview of public goods funding works.
+sidebar_label: How it works
+sidebar_position: 2
+---
+
+Are you interested in running a funding round in your community? 🧐
+
+You’re on the right page! 🥳 We aim to provide guidance and an overview of the process involved.
+
+Besides a few technical & operational requirements, we are very open to new ideas of running rounds. If you have a new idea, feel free to share it with us!
+
+## What is Quadratic Funding?
+
+Quadratic Funding (QF) empowers communities to allocate resources towards funding Public Goods. For those who don't know much about Quadratic Funding, [start here](what-is-quadratic-funding.md).
+
+# Process overview
+
+Here’s an overview of setting up & running a funding round:
+
+1. Apply
+2. Prepare
+3. Run the round
+4. Retrospective
+
+## Timelines
+
+Timelines of running a QF round varies widely depending on the structure of the round, as well as your team’s bandwidth to organise, promote, deploy & operate the round.
+
+We typically recommend **8-10 weeks** of lead time to prepare to run a QF round.
+
+## Time commitment
+
+Requirements here vary widely depending on the structure of the round & team size.
+
+This includes technical & operational work:
+
+- **Technical work**
+  Eg. Configuring wallets, deploying contracts & deploying the website
+  We’d recommend allocating approximately 200 hours of developer time to prepare for a round.
+- **Operational work**
+  Marketing, communications, curating & reviewing project applications & community support
```

Comment:
<img width="843" alt="image" src="https://github.com/privacy-scaling-explorations/maci/assets/9023842/a1ad0a7e-e815-4141-99e5-ab9d81ae8215">

⬆️ looks like this, maybe adding a period or a colon after topic?